streams/streams_extra/run_stream

```diff
@@ -195,6 +195,13 @@ build_images()
 setup_sizing()
 {
 	numa_nodes=`lscpu | grep NUMA | grep CPU | wc -l`
+	if [ $numa_nodes -le 0 ];then
+		numa_nodes=1
+	fi
+
+	if [ -z $numa_nodes ] || [ $numa_nodes -le 0 ]; then
```

Comment:
Line 202, [ $numa_nodes -le 0 ] will never occur, because lines 198-200 take care of it.  Delete the second check.